### DISCUSSION TITLE
Barrel or endcap? DD4hep plugin

### DISCUSSION THREAD

**Discussion by paulgessinger** [acts-project/acts#630] [Q&A] [answered]
Originally posted as an issue #430 by @whit2333. We thought we'd try out the discussions section for this type of issue.

---

What is the best way to build the dd4hep ACTS extension for the angled tracking detectors shown here?

![barrel_endcap](https://user-images.githubusercontent.com/2065918/92024567-f64b7500-ed23-11ea-84b3-68de3501acb0.png)

**Comment by paulgessinger** [comment]
Maybe @asalzburger can re-comment as sort of an 'answer'?

**Comment by asalzburger** [comment]
We had sorted that out with individual layers, actually.

**Reply by whit2333** [reply]
Not the ideal solution. The plugin should be a bit smarter in parsing the dd4hep geometry and choosing the order of construction.

**Comment by asalzburger** [comment, marked as answer]
Let's close this one as the DD4hep plugin will undergo changes.
